### DISCUSSION TITLE
Welcome to libpysal Discussions!

### DISCUSSION THREAD

**Discussion by jGaboardi** [pysal/libpysal#689] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by snehaseveriya23** [comment]
Hi everyone,

I am Sneha, a 2nd year undergraduate student pursuing BTECH in Data Science Engineering. I’ve recently started contributing to libpysal docs and have been looking into the shift from spatial weights to the Graph framework. While working on a small docs PR, I also came across the broader Graph documentation work in https://github.com/pysal/libpysal/pull/854.

I’d like to continue helping with Graph-related documentation, especially from a beginner’s perspective. Are there any specific areas where additional clarity or examples would be most useful right now?

Thanks!
